h/services/annotation_stats.py

```diff
@@ -48,6 +49,24 @@ def group_annotation_count(self, pubid):
         params = MultiDict({"limit": 0, "group": pubid})
         return self._search(params)
 
+    def total_group_annotation_count(self, pubid, unshared=True):
+        """
+        Return the count of all annotations for a group.
+
+        This counts all of the group's annotations and replies from all users.
+
+        If `unshared=True` then "Only Me" annotations and replies in the group
```

Comment:
I had trouble passing this parameter meaning.

 It might be clearer with a more verbose name? `include_only_me`? Not sure